package-examples/wordpress/README.md

```diff
@@ -1,92 +1,85 @@
-          └── [mysql-deployment.yaml]  Service wordpress-mysql
-  
-  The fetched package contains [setters]. Invoke [list-setters] command to list
-  the [setters] recursively in all the packages.
-  
-  $ kpt cfg list-setters wordpress/
-  
-      wordpress/
-               NAME             VALUE      SET BY   DESCRIPTION   COUNT   REQUIRED  
-        gcloud.core.project   PROJECT_ID                          3       No        
-        image                 wordpress                           1       No        
-        tag                   4.8                                 1       No        
-        teamname              YOURTEAM                            3       Yes       
-      
-      wordpress/mysql/
-               NAME             VALUE      SET BY   DESCRIPTION   COUNT   REQUIRED  
-        gcloud.core.project   PROJECT_ID                          3       No        
-        image                 mysql                               1       No        
-        tag                   5.6                                 1       No        
-        teamname              YOURTEAM                            3       Yes       
-        
-   You may notice that the [auto-setter] `gcloud.core.project` is already set if you
-   have `gcloud` configured on your local.
-  
-  ### Provide the setter values
-  Provide the values for all the [required setters]. By default, [set] 
-  command is performed only on the resource files of provided package and not its 
-  subpackages. `--recurse-subpackages(-R)` can be leveraged to run the command on 
-  subpackages recursively.
-  
-  $ kpt cfg set wordpress/ teamname myteam -R
-  
-      wordpress/
-      set 3 field(s)
-      
-      wordpress/mysql/
-      set 3 field(s)
-  
-  ### Apply the package
-  
-  Apply all the contents of the package recursively to the cluster
-  
-  $ kubectl apply -f wordpress/ -R
+          ├── [deployment.yaml]  PersistentVolumeClaim mysql-pv-claim
+          ├── [deployment.yaml]  Deployment wordpress-mysql
+          └── [deployment.yaml]  Service wordpress-mysql
+
+### Configure namespace
+
+By default, these packages will be deployed into `default` namespace. Provide a namespace by
+adding [set-namespace] function to the pipeline definition in `wordpress/Kptfile`.
+
+    pipeline:
+      mutators:
+      - image: gcr.io/kpt-fn/set-namespace:v0
+        configMap:
+          namespace: my-space
+
+### Configure setter values
+
+Setters are listed under `apply-setters` function in the pipeline definition of each package.
+You may declare new desired values for the setters by editing the `Kptfile` directly.
+
+### Render the declared values
+
+    $ kpt fn render wordpress/
+
+### Apply the package
+
+Apply all the contents of the package recursively to the cluster
+
+    $ kpt live init wordpress/
+
+      namespace: default is used for inventory object
+      Initialized: wordpress/inventory-template.yaml
+
+    $ kubectl create ns my-space
```

Comment:
do we want to have these steps here?  Namespace is good hygiene but if we want to do that maybe what we should do is either provide it in the package or just skip it.

Reply:
The aim of this example is to set common namespace for both packages and apply respective setters for individual packages. So namespace is needed here. Other examples might not need them. They can just be deployed in default namespace.